lib/fog/storage/backblaze.rb

```diff
@@ -6,7 +6,7 @@ class Fog::Storage::Backblaze < Fog::Service
   requires :b2_account_id, :b2_account_token
   recognizes :b2_bucket_name, :b2_bucket_id, :token_cache, :logger
 
-  model_path 'fog/backblaze/models'
+  model_path 'fog/storage/backblaze/models'
```

Comment:
I just moved this files to reflect the naming. Since your classes name were `Fog::Storage::Backblaze` it makes sense to look for them at `fog/storage/backblaze`. If you want them all inside `fog/backblaze` then switch the order of the names. This will help newcomers to get to know your gem easier.